man/systemd-boot.xml

```diff
@@ -214,6 +214,17 @@
       </varlistentry>
     </variablelist>
 
+    <para>The boot menu is shown when a non-zero menu timeout has been configured. If the menu timeout has
+    been set to zero, it is sufficient to press any key — before the boot loader initializes — to bring up
```

Comment:
It's still not quite clear - should the user press and hold the key, or just pressing it once is enough? Or repeatedly, to hit the timeframe?